controllers/users.js

```diff
@@ -276,18 +277,78 @@ const updateSelf = async (req, res) => {
  * @param res {Object} - Express response object
  */
 const postUserPicture = async (req, res) => {
+  const { file } = req;
+  const { id: userId, discordId } = req.userData;
```

Comment:
correct me if I am incorrect not all users in the database have `discordId`. what if the field is missing in the user document. we should handle that as well.

Reply:
There is a middleware that checks if the user is discord verified or not. But I think we need to put it in this route. Good catch actually